wgsl/index.bs

```diff
@@ -1780,6 +1780,18 @@ The originating variable for a formal parameter of a function depends on the
 [=call site|call sites=] for the function.
 Different call sites may supply pointers into different originating variables.
 
+If a reference or pointer access is out of bounds, an <dfn noexport>invalid
+memory reference</dfn> is produced.
+[=Load Rule|Loads=] from an invalid reference return one of:
+    * a value from any [=memory locations|memory location=] of the [=originating variable=]
+    * the zero value for store type of the reference
+    * if the loaded value is a vector, the value (0, 0, 0, x), where x is:
```

Comment:
may need a bit of clarify (possibly in follow-up) on, like, what if it's `vec2`. Maybe we should just say here "if the loaded value is `vec4`?